<a href="https://colab.research.google.com/github/JEduardoDimasR/Proyecto-personal/blob/main/Pron%C3%B3stico_de_transacciones_en_una_cadena_de_tiendas_minoristas.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

## Cargar librerías

In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
from statsmodels.tsa.arima_model import ARIMA
from statsmodels.tsa.stattools import adfuller
from statsmodels.tsa.seasonal import seasonal_decompose
from sklearn.metrics import mean_squared_error, mean_absolute_percentage_error
from sklearn.preprocessing import StandardScaler
from keras.models import Sequential
from keras.layers import GRU, Dense

## Cargar datos

In [3]:
store_df = pd.read_excel("store.xls")

## Comprender la información de los datos

In [4]:
store_df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 115 entries, 0 to 114
Data columns (total 2 columns):
 #   Column        Non-Null Count  Dtype         
---  ------        --------------  -----         
 0   Date          115 non-null    datetime64[ns]
 1   Transactions  115 non-null    int64         
dtypes: datetime64[ns](1), int64(1)
memory usage: 1.9 KB


In [7]:
store_df.shape

(115, 2)

Se cuenta con un conjunto de datos de 115 observaciones y 2 variables, una de tipo númerica y otra de tipo temporal.
